src/main/java/vazkii/botania/common/item/rod/ItemTerraformRod.java

```diff
@@ -127,9 +127,7 @@ private void terraform(ItemStack stack, Level world, Player player) {
 			}
 
 			if (!blocks.isEmpty()) {
-				for (int i = 0; i < 10; i++) {
-					world.playSound(player, player.getX(), player.getY(), player.getZ(), SoundEvents.SAND_STEP, SoundSource.BLOCKS, 1F, 0.4F);
-				}
+				world.playSound(player, player.getX(), player.getY(), player.getZ(), ModSounds.terraformRod, SoundSource.BLOCKS, 1F, 1F);
```

Comment:
This sound is still much more than loud enough, even if it's not played 10 times. I think it sounds the same when played 10x anyways